README.md

```diff
@@ -6,37 +10,35 @@
 [![commits last month](https://img.shields.io/github/commit-activity/m/illinoisdata/ElasticNotebook)](htps://github.com/illinoisdata/kishu)
 [![GitHub stars](https://img.shields.io/github/stars/illinoisdata/ElasticNotebook)](htps://github.com/illinoisdata/kishu)
 --->
+</div>
 
-# Kishu: Versioned and Undoable Notebook System
+Kishu is a system for versioning notebook session states in IPython-based notebooks (e.g. JupyterLab, Jupyter Hub). It efficiently and transparently checkpoints both variable and code states of notebook sessions, enabling users to undo cell executions and manage branching states containing objects such as ML models, plots, and dataframes via Git-like commits and checkouts.
 
-<br>
 <div align="center">
-<img width="350" src="docs/images/kishu_with_background.png"/>
+<img width="960" src="docs/images/kishu_frontpage.png"/>
```

Comment:
Zoomed-in success notification is blurry. Please make resolution consistent across the image. Alternatively consider not showing the zoomed-in to be consistent with the "Undo Execution" on top